APISpec.md

```diff
@@ -0,0 +1,144 @@
+# Todo API Spec
```

Comment:
You have users in your API which indicates a sense of ownership around a user's todo and since this is a multi-tenant setup we should probably prevent users from being able to edit or view other users' TODOs

Reply:
I added the authorization header to requests that I think need authentication